packages/client/src/services/user-account-service.ts

```diff
@@ -1,14 +1,13 @@
-import { AuthRequestDto } from 'lebkuchen-fm-service';
 import { redirectTo } from './redirect-to';
 
 export function userLogin(username: string, password: string) {
-  const data: AuthRequestDto = { username, password };
+  const body = new URLSearchParams();
```

Comment:
shouldnt we use `FormData` instead of `URLSearchParams`?

Reply:
Actually - no. While the new backend accepts both `application/x-www-form-urlencoded` and `multipart/form-data` the old backend (after this change) can only support `application/x-www-form-urlencoded` (without much work). 
Besides - for simple forms like this one we [should be using `application/x-www-form-urlencoded` anyway](https://stackoverflow.com/questions/4007969/application-x-www-form-urlencoded-or-multipart-form-data).